source/common/tracing/zipkin_tracer_impl.cc

```diff
@@ -0,0 +1,201 @@
+#include "common/tracing/zipkin_tracer_impl.h"
```

Comment:
The LS code should really be in a LS namespace. There is no reason for this code to be outside the namespace of all the other Zipkin code. Please move it.